openwisp_notifications/templates/emails/batch_email.txt

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}
+
+[{{ site_name }}] {{ notifications_count }} {% translate "new notifications since" %} {{ start_time }}
+
+{% for notification in notifications %}
+- {{ notification.message }}
+  {% translate "URL" %}: {{ notification.url }}
+  {% translate "Timestamp" %}: {{ notification.timestamp|date:"F j, Y, g:i a" }}
+  {% if show_notification_description %}
+      {% translate "Description" %}: {{ notification.rendered_description }}
```

Comment:
The description line should be shown after the message and only if not empty, without indentation. Let's always show the description if not empty, regardless of the amount of notifications.